main.py

```diff
@@ -45,16 +48,21 @@ async def listen_to_grpc_stream(
 
     try:
         stub = QueryStub(channel)
-        request = StreamOrderbookUpdatesRequest(clob_pair_id=clob_pair_ids)
+        # parse subaccount ids, each subaccount id is string with format owner/number
+        subaccount_protos = [SubaccountId(owner=sa.split('/')[0], number=int(sa.split('/')[1])) for sa in subaccount_ids]
+        request = StreamOrderbookUpdatesRequest(clob_pair_id=clob_pair_ids, subaccount_ids=subaccount_protos)
         with open(log_path, 'w') as log:
             async for response in stub.StreamOrderbookUpdates(request):
                 # Log the message
+                print('response', response)
```

Comment:
extra print, if you need to print use logging library?

Reply:
oops, removed